js/jquery.tooltip.js

```diff
@@ -155,12 +155,22 @@
 		}
 		if (!this.tooltip || !this.tooltip.length) return;
 
-		if (this.tooltip.is(':hidden')) {
-			this.tooltip.stop(true).fadeTo(0, 0).show();
-			setTimeout(function () { self.align.call(self, self.currentTarget); }, 10);
-			this.align(this.currentTarget).delay(this.conf.showDelay).fadeTo(this.animSpeed, 1);
+		if($('img', this.tooltip).size()){
```

Comment:
`$('img', this.tooltip)` should be cached (but preferably a different approach should be used)